js/file-upload.js

```diff
@@ -32,7 +33,7 @@
 		},
 
 		// Initializes plupload using code from wp.Uploader (wp-includes/js/plupload/wp-plupload.js)
-		initUploader: function () {
+		initUploader: function ( $this ) {
```

Comment:
Hình như ko cần phải refer đến biến `$this` đâu e ạ, chỉ cần `this.$el` là được. E thử lại xem.